dvp_waypoint_replanner:
  # IP and port to bind and listen for incoming velocity profiles
  server_ip: 127.0.0.1
  server_port: 1551

  # Max speed of all waypoints (km/h)
  max_speed: 100.0

dvp_tracked_objects:
  # IP and port to send tracked objects to
  destination_ip: 127.0.0.1
  destination_port: 1552

  # The +/- angle from the vehicle heading to include objects (rad)
  cutoff_angle: 0.7854

  # Objects with a classification score below the threshold will be excluded (0 - 1)
  score_threshold: 0.8
